PhpStorm Protocol (Win)/run_editor.js

```diff
@@ -22,6 +27,10 @@ if (match) {
 		file_system = new ActiveXObject('Scripting.FileSystemObject'),
 		file = decodeURIComponent(match[1]).replace(/\+/g, ' '),
 		search_path = file.replace(/\//g, '\\');
+		
+	if (settings.project_basepath != '' && settings.project_pathalias != '') {
+		file = file.replace(settings.project_basepath, settings.project_pathalias);

```

Comment:
Also I'm not sure if you're aware of it, but PhpStorm does support relative paths. This way if you just replace `project_basepath` with an empty string and PhpStorm has that project opened then it will open the file properly.